arango/collections/base.py

```diff
@@ -689,18 +736,24 @@ def find_near(self, latitude, longitude, limit=None):
         :param limit: the max number of documents to return
         :type limit: int
         :returns: the document cursor
-        :rtype: arango.cursor.Cursor
+        :rtype: arango.cursor.BaseCursor
         :raises arango.exceptions.DocumentGetError: if the documents
             cannot be fetched from the collection
 
         .. note::
             A geo index must be defined in the collection for this method to
             be used
         """
+
+        if limit is None:
+            limit_string = ""
```

Comment:
I tend to use double quotes because they are consistent Unicode python 2->3, whereas single quotes are byte strings in python 2 and Unicode in 3, but I can switch to single if you’d like

Reply:
Oh hmm I didn't know that (good to know). You can leave them as they are then.

Reply:
Actually, perhaps I'm wrong on this one? Everywhere seems to indicate that they're exactly the same, but I remember having handling issues when working with lxml...

Anyway, for what we're doing, the difference shouldn't matter all that much since all of our constants are ascii.  I'll use single quotes.